src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -0,0 +1,267 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Serilog.Events;
+
+#if NETSTANDARD
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Abstractions;
+#else
+using System.Web;
+using System.Security.Principal;
+#endif
+
+namespace Elastic.CommonSchema.Serilog
+{
+    /// <summary>
+    /// Elastic Common Schema converter for LogEvent
+    /// </summary>
+    public class LogEventConverter
+    {
+        public static Base ConvertToEcs(LogEvent logEvent, IECSJsonFormatterConfiguration configuration)
+        {
+            var exceptions = logEvent.Exception != null
+                                ? new List<Exception>{ logEvent.Exception }
+                                : new List<Exception>();
+
+            if (configuration.MapHttpAdapter != null)
+            {
+                exceptions.AddRange(configuration.MapHttpAdapter.Exceptions);
+            }
```

Comment:
noted this as well in my pr #6, `logEvent` itself has quite a few enrichment as well.

https://github.com/serilog/serilog/wiki/Enrichment

These all end up inside `logEvent.Properties`, we can move the known ones to ECS and the unknown ones to `_metadata`